planty/users/views.py

```diff
@@ -16,3 +18,39 @@ def register(request):
     else:
         form = UserRegisterForm()
     return render(request, 'users/register.html', {'form': form})
+
+
+@login_required
+def profile(request, pk):
```

Comment:
I would rename this to `view_profile`. I would also rename the template and the named url in urls.py